src/vulkan/image.cc

```diff
@@ -122,9 +122,9 @@ void Image::Shutdown() {
   memory_ = VK_NULL_HANDLE;
 }
 
-Result Image::CopyToHost(VkCommandBuffer command) {
+void Image::CopyToCPU(VkCommandBuffer command) {
   if (is_image_host_accessible_)
-    return {};
+    return;
```

Comment:
Why isn't this an error if we can't access the image, something has gone wrong, no?